internal/manifest_list_v2.go

```diff
@@ -0,0 +1,100 @@
+package internal
+
+import (
+	"github.com/hamba/avro/v2"
+)
+
+func MustNewManifestListV2Schema() avro.Schema {
```

Comment:
since this is a static schema, can we cache it rather than building it on demand?